source/__tests__/integration/create-resource.test.js

```diff
@@ -67,15 +72,80 @@ describe('Create profile resource template requirements', () => {
           author: "Me",
           title: "My profile",
           resourceId: "my:resource",
-          resourceURI: "http://www.example.com"
+          resourceURI: "http://id.loc.gov"
         })
       .catch(error => console.log(`promise error for filling profile form: ${error}`))
     await page.waitFor(1000) // waiting for .toFillForm(), as resourceURI field does a check
+    page
+      .waitForSelector('input#resourceURI.ng-valid-url')
+      .catch(error => console.log(`promise error checkURL: ${error}`))
     await page.click(exportButtonSel)
     page
       .waitForSelector(alertBoxSel)
       .catch(error => console.log(`promise error for alert box selector: ${error}`))
     await expect_value_in_sel_textContent(alertBoxSel, "my:resource must have at least one property template")
+
+
+    })
+
+    it('requires a valid property uri', async () => {
+      expect(page)
+      .toFillForm('form.sinopia-profile-form[name="profileForm"]', {
+          // all the other required fields from resource template
+          id: "my:profile",
+          description: "Profile description",
+          author: "Me",
+          title: "My profile",
+          resourceId: "my:resource",
+          resourceURI: "http://id.loc.gov"
+      })
+      .catch(error => console.log(`promise error for filling profile form: ${error}`))
+
+      await page.click('a.propertyLink')
+      await page.waitFor(1000)
+      await page.evaluate(() => {
+          let dom = document.querySelector('a.propertyLink');
+          dom.innerHTML = "h";
+      });
+
+      const phtml = await page.evaluate(() => document.querySelector('a.propertyLink').innerHTML);
+      console.log(phtml)
+
+      console.log("PropertyURI Invalid")
+      page
+          .waitForSelector('input#propertyURI.ng-invalid-url')
+          .catch(error => console.log(`promise error checkPropertyURL: ${error}`))
```

Comment:
this test has no "expect" in it anywhere so it will always pass, correct?  I think some of the other new test code has this problem as well.